docs/user-guide/ingress-path-matching.md

```diff
@@ -109,7 +109,7 @@ The following request URI's would match the corresponding location blocks:
 
 ## Warning
 
-The following example describes a case that may inflict unwanted path matching behaviour.
+The following example describes a case that may inflict unwanted path matching behavior.
```

Comment:
Switching to American spelling? `behaviour` is correct as well, just not preferred in America: https://www.grammar.com/behavior_vs._behaviour

Reply:
Good point. I found this running `make misspell`